blog/views.py

```diff
@@ -24,4 +25,7 @@ class EntriesView(ListView):
     template_name = 'blog/blog.html'
     model = Entry
 
+class UpdateEntryView(UpdateView):
```

Comment:
This class in complete.  UpdateView's require forms to be created for them and assigned as:

```
form_class = MyFormClass
```

Looks like you still need to create the form for this update view.  See: 

- https://docs.djangoproject.com/en/1.8/topics/forms/modelforms/